client/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -e
+
+case "$1" in
+    develop)
+        echo "Running web application in development mode"
+        exec yarn dev
+        ;;
+    test)
+        echo "Running Tests"
+        exec yarn cypress:run
+        ;;
+    start)
+        echo "Running web application in production mode"
+        exec yarn start:prod
+        ;;
+    *)
+        echo "Usage: entrypoint.sh {develop|test|start:prod}" >&2
```

Comment:
start:prod -> start

Reply:
`start` needs a previous build to run prod mode, which is not present when container runs
Thats why I added a [new script](https://github.com/Vizzuality/landgriffon/pull/24/files#diff-1846122c2c83a486a3693f7966aa522c34cf489f674185c4da0d9221683fd81f) in `/client/package.json`

Reply:
To me is fine, but you can also add another line only for the build